.github/workflows/release.yml

```
@@ -0,0 +1,62 @@
+name: releaseing
+
+on:
+  push:
+    branches:
+      - release/major
+      - release/minor
+      - release/patch
+      - release/premajor
+      - release/preminor
+      - release/prepatch
+      - release/prerelease
+
+jobs:
+  releaseing:
+    name: releaseing
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+      - uses: actions/cache@v2
```

```suggestion
      - uses: actions/cache@v3
```
IMO. こちらもv3でも良いかもです！